clang-tools-extra/clangd/SystemIncludeExtractor.cpp

```diff
@@ -203,6 +214,7 @@ template <> struct DenseMapInfo<DriverArgs> {
         Val.Driver,
         Val.StandardIncludes,
         Val.StandardCXXIncludes,
+        Val.Stdlib,
```

Comment:
nit: it looks like the previous patch which introduced `Target` forgot to add it here, could you add please?

Reply:
Done, and resorted these to match the order in `DriverArgs` definition so it'll be easier to tell when things are missing in the future.